src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -344,4 +344,27 @@ public function filter_add_body_classes( $classes ) {
 
 		return $classes;
 	}
+
+	/**
+	 * Filter the template file in case we're in single event
+	 * and we need to use the theme overrides.
+	 *
+	 * @since  TBD
+	 *
+	 * @return string
+	 */
+	public function filter_template_file( $file, $name, $template ) {
+		
+		if ( 
+			is_singular( TEC::POSTTYPE )
+			&& 'single-event' === tribe_context()->get( 'view' )
+		) {
+			$override = V1_Event_Templates::getTemplateHierarchy( 'default-template' );
+			if ( ! empty( $override ) ) {
+				$file = $override;
+			}
+		}		
```

Comment:
:no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed

<sub>hash: faf7aa15b3b6482b2f09a27bc1bacae4</sub>